src/helpers.jl

```diff
@@ -5,7 +5,12 @@ function count_rows(data)::Integer
     if Tables.rowaccess(data)
         return length(Tables.rows(data))
     else
-        @error "Can't count the number of rows"
+        # Because we wrapped in
```

Comment:
Hmmmm, this function isn't going to work in all cases. For example, the result of `Tables.rows` isn't required to support `Base.haslength`. The only way to do this reliably (currently) is you can call `Tables.rowcount` on the result of `Tables.columns`. That's because the object returned from `Tables.columns` is required to have columns with a given length.

Reply:
Thanks for your help with this, I was only reading the Tables.jl documentation and wouldn't have known about `rowcount`.

Similar to another issue you raised, this seems important and enough and simple enough to fix that I'll merge this PR and open a quick one to fix it.